drivers/timer/nrf_rtc_timer.c

```diff
@@ -211,6 +211,24 @@ void _timer_idle_enter(s32_t sys_ticks)
 
 
 #ifdef CONFIG_TICKLESS_KERNEL
+/*
+ * Set RTC Counter Compare (CC) register to max value.
+ */
```

Comment:
This function also updates _sys_clock_tick_count, which is not mentioned in the comment. Moreover as this is something different than programming CC register, _sys_clock_tick_count update should be moved away from this function.